src_new/adapters/prebid.js

```diff
@@ -229,6 +303,17 @@ function generatedKeyCallback(adapterID, adUnits, adapterConfig, impressionID, g
 			bids: [],
 			divID : divID
 		};
+	}else if(CONFIG.getSingleImpressionSetting()){
+		var flag = false;
+		adUnits[code].bids.forEach(function(element) {
+			if(element.bidder == adapterID){
+				flag= true;
+				return;
```

Comment:
the return statement here can be removed, it is not adding any value.
Following code prints all values of the array even if you have put return statement:
[1, 2, 3, 4].forEach(function(a){console.log(a); return;})

Reply:
Done